### PR TITLE
06112020 blog post

### PR DESCRIPTION
**Pull request by PatrickGoRaft**

### CODE REVIEW

File: _posts/2020-06-11-tech-community-diversity.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+There are black people in the tech community who are struggling to find their voice, struggling to find acceptance, and struggling just to exist in their fields. Reach out to them, ask them how they’re doing and how you can help.  Don't consider yourself to be pandering or trying to capitalize on a popular issue.  There have been countless sacrifices made to reach this time of change and, now that we're here, we should do something about it. We’re here now and we should do something about it.
```
Comment (omnipresent07) [inline _posts/2020-06-11-tech-community-diversity.md:40]: The last line here seems like a repeat: "We’re here now and we should do something about it." should be deleted.

File: _posts/2020-06-11-tech-community-diversity.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+title: "Diversity in the Tech Industry"
+short_title: "Diversity in Tech"
+date: 2020-06-11
+categories:
+featimg: /assets/images/featimage/Diversity-Small.jpg 
+bgimg: /assets/images/bgimg/blm-banner.jpg 
+author: Patrick  
+---
+<img class='float-img-right' src="/assets/images/featimage/Patrick.jpg"  alt="Patrick A">
+When I look at myself in the mirror, I see many things. I see a software developer, I see an educator, I see someone with a master in Computer Science. However, from the time I was a child to today and into the future, I also see a black man living in America.
```
Comment (meissadia) [inline _posts/2020-06-11-tech-community-diversity.md:12]:
```suggestion
When I look at myself in the mirror, I see many things. I see a software developer, I see an educator, I see someone with a master's in Computer Science. However, from the time I was a child to today and into the future, I also see a black man living in America.  
```

File: css/main.scss [hunk 3]
```diff
@@ -57,7 +57,7 @@ $xlarge: 1200px;
 }
 
 #blog-content img {
-  height: 500px;
+  max-height: 500px;
   width: 500px;
```
Comment (meissadia) [inline css/main.scss:61]:
```suggestion
```

File: _posts/2020-06-11-tech-community-diversity.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+There are black people in the tech community who are struggling to find their voice, struggling to find acceptance, and struggling just to exist in their fields. Reach out to them, ask them how they’re doing and how you can help.  Don't consider yourself to be pandering or trying to capitalize on a popular issue.  There have been countless sacrifices made to reach this time of change and, now that we're here, we should do something about it. We’re here now and we should do something about it.
```
Comment (meissadia) [inline _posts/2020-06-11-tech-community-diversity.md:34]:
```suggestion
There are black people in the tech community who are struggling to find their voice, struggling to find acceptance, and struggling just to exist in their fields. Reach out to them, ask them how they’re doing and how you can help.  Don't consider yourself to be pandering or trying to capitalize on a popular issue.  There have been countless sacrifices made to reach this time of change and, now that we're here, we should do something about it.
```

**Pull request merged**